README.md

```diff
@@ -24,6 +24,8 @@
 
 This component creates a connection to an SFTP server to read and upload files.
 
+The optional environment variable `MAX_FILE_SIZE` should be set in settings to provide the maximum file size that can be uploaded in **megabytes (mb)**. The default value for `MAX_FILE_SIZE` is 100MB.
```

Comment:
@hannah-bulmer why did you decide to use 100MB? Did you test such attachment size on the platform?
@stas-fomenko as I know currently it is something like 10MB?

Reply:
Tigran updated the platform documentation from 10MB to 100MB a couple of days ago: https://github.com/elasticio/elasticio.github.io/pull/438 We should double check to make sure that 100 MB works correctly.

Reply:
`SFTP: Download file by name` cannot get ~97mb file with `EIO_REQUIRED_RAM_MB` = 4000, 
Component throws an error - `Error: Component did not send any data during timeout.`

Actually I thought that limit is 10mb - in this case test cases passes

10+MB files usually uploaded incorrectly even with additional memo, so if actual limit is 100MB then SFTP component has a bug with handling big (<100 mb) files